recipes/featuretools_sql/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "featuretools_sql" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/featuretools_sql-{{ version }}.tar.gz
+  sha256: 46d090103f7721958a940a10b1a7a40a385278b802134f9ca04ed90d46a45291
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6,<4.0
+    - pip
+  run:
+    - python >=3.6,<4.0
+    - numpy >=1.17.5
+    - pandas >=1.3.0
+    - featuretools >=1.5.0
+    - psycopg2 >=2.9.3
+    - connectorx >=0.3.0
+    - snowflake-sqlalchemy >=1.4.2
+
+test:
+  imports:
+    - featuretools_sql
```

Comment:
```suggestion
    - featuretools_sql
    - pytest --pyargs featuretools_sql
```

Reply:
these would require the user to have set up `mysql` and `postgres` services